boards/arm/stm32wb5mm_dk/doc/stm32wb5mm_dk.rst

```diff
@@ -0,0 +1,226 @@
+.. _stm32wb5mm_dk_discovery_kit:
+
+ST STM32WB5MM-DK
+################
+
+Overview
+********
+
+The STM32WB5MM-DK Discovery kit is designed as a complete demonstration
+and development platform for the STMicroelectronics STM32W5MMG module based
+on the Arm |reg| Cortex |reg|-M4 and Arm |reg|  Cortex |reg|-M0+ cores. The
+STM32 device is a multi-protocol wireless and ultra-low-power device
+embedding a powerful and ultra-low-power radio compliant with the
+Bluetooth |reg| Low Energy (BLE) SIG specification v5.2 and with
+IEEE 802.15.4-2011.
+
+
+STM32WB5MM-DK supports the following features:
+
+* STM32WB5MMG (1-Mbyte Flash memory, 256-Kbyte SRAM)
+    - Dual-core 32‑bit (Arm |reg| Cortex |reg|-M4 and M0+)
```

Comment:
check indentation in this bullet list so that it does not render oddly (right now it's a "definition list")